docs/index.html

```diff
@@ -0,0 +1,266 @@
+
+
+<!--
+Documentation generated by Skydoc
```

Comment:
Why check in generated stuff?

Reply:
1) GitHub can serve this at bazelbuild.github.io/rules_nodejs - docs/ folder is a special case
2) by checking in, the docs are versioned with the code, then you can browse the docs at the same tag you depend on